airflow/jobs.py

```diff
@@ -1774,8 +1774,11 @@ def _execute(self):
 
         # create dag runs
         dr_start_date = start_date or min([t.start_date for t in self.dag.tasks])
-        next_run_date = self.dag.normalize_schedule(dr_start_date)
         end_date = end_date or datetime.now()
+        # next run date for a subdag isn't relevant (schedule_interval for subdags
+        # is ignored) so we use the dag run's start date in the case of a subdag
+        next_run_date = self.dag.normalize_schedule(dr_start_date) \
+            if "." not in self.dag.dag_id else dr_start_date
```

Comment:
please use () across multiple lines and use "dag.is_subdag" instead of manually checking